internal/ingress/controller/store/store.go

```diff
@@ -165,6 +165,7 @@ func (i *Informer) Run(stopCh chan struct{}) {
 		i.Service.HasSynced,
 		i.Secret.HasSynced,
 		i.ConfigMap.HasSynced,
+		i.Pod.HasSynced,
```

Comment:
Was this a bug? How did you find our about this?

Reply:
> Was this a bug? How did you find our about this?

Yes, while checking if we wait for all the informers :bug: